### DISCUSSION TITLE
How to pass watchtower notifications in ifttt json payload via shoutrrr

### DISCUSSION THREAD

**Discussion by milindpatel63** [containrrr/shoutrrr#209] [Q&A] [answered]
How do i use the ifttt service of shoutrrr to send watchtower notifications to a ifttt webhook?
Specifically, how do i pass the log in the webhook....
i can successfully send the webhook using this ifttt url
ifttt://key/?events=watchtower&titlevalue=1&messagevalue=2
i also need to send the log message as either of value1,2 or 3 in ifttt webhook....how do i do that?

**Comment by jvonhoff** [comment, marked as answer]
Found your post, and figured it out.  I added the following to my docker-compose:
```
WATCHTOWER_NOTIFICATIONS=shoutrrr
WATCHTOWER_NOTIFICATION_URL=ifttt://API_TOKEN/?events=EVENT&value1=Watchtower
```
First, visit [this IFTTT page](https://ifttt.com/maker_webhooks). Then, click Documentation, and **API_TOKEN** is at the top of the page where it says "Your key is:".  For me, this opened in a new tab.
Now, you can close this new tab, or go back to the previous page.  Next, click the My Applets just below Documentation, and click on the webhook you're trying to use.  Near the top, click on Settings (with the gear icon) and now it will show the IF and THEN. The IF should be your webhook, so click on Edit, and **EVENT** is noted in the text box.  Mine is called "curled" because I was using curl to call it in some bash scripts.

Hope that helps!
